twitchio/models.py

```diff
@@ -530,3 +530,38 @@ def __init__(self, data: dict):
 
     def __repr__(self):
         return f"<WebhookSubscription callback={self.callback} topic={self.topic} expires_at={self.expires_at}>"
+
+
+class Stream:
+
+    __slots__ = (
+        "id",
+        "user",
+        "game_id",
+        "game_name",
+        "type",
+        "title",
+        "viewer_count",
+        "started_at",
+        "language",
+        "thumbnail_url",
+        "tag_ids",
+        "is_mature",
+    )
+
+    def __init__(self, http: "TwitchHTTP", data: dict):
```

Comment:
You've made all of these attributes tuples, and please turn parameters that can be turned into ints into such

Reply:
Strange they all became tuples. I wonder if black formatter did that at one point. Should be correct now. Thanks for the feedback!